testdata/lava.yaml

```diff
@@ -0,0 +1,8 @@
+lava: v1.0.0
+checktypesURLs:
+  - checktypes.json
+targets:
+  - identifier: ..
```

Comment:
Why is the identifier the parent folder?

Reply:
Because paths are relative to the location of the config file. In this case we want to point to the root of the repository which is one directory up.